docker-storage-setup.sh

```diff
@@ -537,6 +537,32 @@ is_block_dev_partition() {
   return 1
 }
 
+check_block_dev_sig() {
+  local bdev=$1
+  local sig
+
+  [ ! -b "$bdev" ] && bdev="/dev/${bdev}"
+
+  if ! sig=$(wipefs -p $bdev); then
+    Fatal "Failed to check signatures on device $bdev"
+  fi
+
+  [ "$sig" == "" ] && return 0
+
+  while IFS=, read offset uuid label type; do
+    [ "$offset" == "# offset" ] && continue
+
+    # Ignore dos signatures for now. We have already checked that disk
+    # does not have partitions. But there might be leftover partition
+    # signature and so far we have not forced user to wipe it. So in an
+    # attempt to not change user visible behavior, don't force user to
+    # wipe dos signature and just make sure disk is not partitioned.
```

Comment:
Right, this should apply to gpt partition table as well. 

So looks like you like the other approach better where if any kind of signatures are present on disk, reject it? 

I am fine with it. I am only concerned that some people will call it regression. They will say that previous version of dss accepted disks with dos/gpt signatures but this new version does not. 

I guess this should not create issues with atomic upgrade, as thin pool is already in place and new disks are not being added. So only thing we have to deal with is telling people to use wipefs with new version of docker-storage-setup and handle the surprise factor.

I can make that change if you like it better.